translations/process/translation-process.md

```diff
@@ -0,0 +1,26 @@
+# Working on translations
```

Comment:
what about https://gitlocalize.com/ ?

Reply:
Looks interesting. I've installed it in my fork, but it allows to only add a translation a block by block, so I haven't migrated my translation into it.

@etolstoy, what do you think about the tool? It looks neat and it's free.

Reply:
Wow, it looks really awesome. Let's try to integrate it!